x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/xcontent/XContentUtils.java

```diff
@@ -14,6 +14,11 @@
 
 public class XContentUtils {
 
+    public static final class AuditToXContentParams {
+        public static final String INCLUDE_CREDENTIALS = "include_credentials";
```

Comment:
Apologies if we have already discuss this in the past. Does this come from a user request? I can't see the use case for auditing credential values in plaintext and it seems like a dangerous option to include ( even if it defaults to false )

Reply:
There was no external influence, it was just my "reflex" to leave it configurable.
But I've gotten the message. No credentials whatsover in the logs, not configurable, at least in this first iteration.